arlo_server/election_settings.py

```diff
@@ -0,0 +1,63 @@
+from flask import jsonify, request
+from jsonschema import validate
+
+from arlo_server import app, db
+from arlo_server.auth import require_audit_admin_for_organization
+from arlo_server.models import Election
+
+from util.jsonschema import nullable, Obj, Str, Bool, Int
+
+
+PUT_ELECTION_SETTINGS_REQUEST_SCHEMA = Obj(
+    electionName=Str,
+    online=Bool,
+    randomSeed=nullable(Str),
+    riskLimit=nullable(Int),
+    state=nullable(Str),
```

Comment:
If you wanted to, could add an enum to validate the `state`

Reply:
I like that. Have the values handy?

Reply:
Good question. I found this: https://www.50states.com/abbreviations.htm

Reply:
🤦‍♂ I took this to mean "audit state" like "ready" or "frozen" or "completed" 🤣

Reply:
Lol, I was wondering. I mean it's not necessarily as simple as the 50 states...

Reply:
@jonahkagan @eventualbuddha :laughing: That is an unfortunate ambiguity. Should we perhaps change the name of `state` to something more explicit?

Also, I'm sending them as the full name of the state like `'Alabama'`, not abbreviations. I could implement it so that it displays the option as the full name but sends back the abbreviation I suppose... depends on where we're displaying it on client-side and which one we want to see I suppose. Thoughts, @mcchilders?

Reply:
I would be inclined to store abbreviations and to have a mapping for display. Otherwise localization becomes very annoying.

Reply:
An amazing and rare case where the government defines the enum for you!

Reply:
Updated.